src/sysutil.rs

```diff
@@ -121,7 +121,8 @@ impl SysUtil {
                         "Found user '{}' with UID '{}' and full name: {gecos}",
                         entry.name, entry.uid
                     );
-                    gecos
```

Comment:
Could you also fix the log message here? Maybe assign the first entry to a variable that's used in the log message, and is also returned.

Reply:
Yes that makes sense. Should be done now.